server/sonar-server/src/main/java/org/sonar/server/platform/ws/InfoAction.java

```diff
@@ -64,12 +69,38 @@ public void handle(Request request, Response response) {
   private void writeJson(JsonWriter json) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "writeJson" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)